packages/expo-image/android/build.gradle

```
@@ -92,4 +92,5 @@ dependencies {
   api "com.squareup.okhttp3:okhttp:${safeExtGet("okHttpVersion", DEFAULT_OKHTTP_VERSION)}"
 
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet('kotlinVersion', '1.4.21')}"
+  implementation "jp.wasabeef:glide-transformations:4.0.0"
```

I see there's a newer version: https://github.com/wasabeef/glide-transformations#gradle. Is there any reason why you choose `4.0.0` over `4.3.0`?
```suggestion
  implementation "jp.wasabeef:glide-transformations:4.3.0"
```